src/libs/nav.test.ts

```diff
@@ -0,0 +1,19 @@
+import { isTerraNavKey, terraNavKey } from './nav';
+
+describe('isTerraNavKey', () => {
+  it('validates good key name', () => {
+    // Act
+    const result = isTerraNavKey(terraNavKey('workspace-dashboard'));
+
+    // Assert
+    expect(result).toBe(true);
```

Comment:
How could this pass given the changes?

Reply:
I think we cleared this up.  Environments nav actions are purposefully named generically, and these terra key names are not changing.